source/fe/mapping_fe.cc

```diff
@@ -32,7 +32,6 @@
 #include <deal.II/grid/tria_iterator.h>
 
 #include <deal.II/lac/full_matrix.h>
-#include <deal.II/lac/tensor_product_matrix.h>
```

Comment:
:+1: I was thinking to investigate this since it was a bit annoying since this file had to recompiled for each modification in `tensor_product_matrix.h`.